### DISCUSSION TITLE
We are reviving Unable to Fetch Commits

### DISCUSSION THREAD

**Discussion by snake-py** [getsentry/support#31] [Support] [answered]
We are unsure why this all of the sudden happens, it was working before - any good way to investigate the issue? The email just says internal server error. 

We were unable to fetch the commit log for your release (app-name@feature-yyyy-xxxxx) for repository orga / project / repo in the organization due to the following error:

An internal error occurred with the integration and the Sentry team has been notified

**Comment by dall2812** [comment]
I second that, same problem here.

**Comment by snake-py** [comment]
After we uninstalled it from the repo and reinstalled it again, the issue seemed to wen away.

**Comment by Kobby-Bawuah** [comment, marked as answer]
Hey @snake-py, github had an outage then. Unfortunately, the only way to resolve this was to uninstall and re-installing the integration. Rest assured that we are taking steps to avoid similar situations in the future. Hey @snake-py, did re-installing it work out for you?

**Comment by snake-py** [comment]
@Kobby-Bawuah sorry to respond so late. The issue went away by itself. I think we uninstalled it and reinstalled it. After a while, it was just gone and worked again.
